tools/cli/adminCommands.go

```diff
@@ -62,3 +65,48 @@ func AdminDescribeWorkflow(c *cli.Context) {
 
 	prettyPrintJSONObject(resp)
 }
+
+// AdminDescribeHistoryHost describes history host
+func AdminDescribeHistoryHost(c *cli.Context) {
+	// using service client instead of cadence.Client because we need to directly pass the json blob as input.
+	serviceClient := getAdminServiceClient(c)
+
+	wid := c.String(FlagWorkflowID)
+	sid := c.String(FlagShardID)
+	addr := c.String(FlagAddress)
```

Comment:
You need a new flag FlagHistoryAddress here to avoid collision of global frontend address and this specific history address.

Reply:
Yeah, that's a good point. I started with different flag but then found a similar one and tried to reuse it. But that won't work because of the conflict.